airflow/providers/google/CHANGELOG.rst

```diff
@@ -23,6 +23,15 @@
 Changelog
 ---------
 
+9.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+* ``Migrate DV360 operators to use API v2 (#30326)``
```

Comment:
Can we say a bit more?
explain what is changed? maybe link to the google relevant migration docs? the goal is to prevent questions of users "what do we need to do"